python/exercise1/vowel_counter.py

```diff
@@ -2,14 +2,14 @@
 
 def num_vowels(text):
     """Return the number of vowels in string."""
-    vowels = "aeiou"
+    vowels = "aeiouy"
     num = 0
     for v in vowels:
         num += text.lower().count(v)        
     return num
 
 def num_consonants(text):
-    vowels = "aeiou"
+    vowels = "aeiouy"
```

Comment:
Have any users requested that `y` be counted as a vowel? I believe it is more often considered a consonant. If we do want to implement support for treating `y` as a vowel, I suggest this be implemented using an optional kwarg e.g. `y_is_vowel: bool`.

Reply:
Suggest rolling back this commit and raising a separate issue for this so it can be discussed separately from the current bug fix.